packages/devtools_app/lib/src/inspector/flutter/inspector_screen_details_tab.dart

```diff
@@ -171,17 +132,7 @@ class _LayoutDetailsTabState extends State<LayoutDetailsTab>
   @override
   Widget build(BuildContext context) {
     super.build(context);
-    // TODO(albertusangga): Visualize non-flex widget constraint model
-    if (root == null || !StoryOfYourFlexWidget.shouldDisplay(selected))
-      return const SizedBox();
-    final flexLayoutProperties = FlexLayoutProperties.fromDiagnostics(root);
-    final highlightChild =
-        selected.isFlex ? null : selected.parent.childrenNow.indexOf(selected);
-    return StoryOfYourFlexWidget(
-      flexLayoutProperties,
-      highlightChild: highlightChild,
-      inspectorController: controller,
-    );
+    return responsibleWidget(selected);
```

Comment:
What does this name mean?

Reply:
The idea here is to make the DetailsTab listen to the inspector selection event and decide which widget is resposnsible for rendering that selected node.
For now we are only supporting Flex-based widget, in the case where the selected node is not a Flex-based widget we are not rendering anything except SizedBox now, but in the future we might support rendering non-Flex based widget, for example for Container based widget we can render bounding box model (padding, margin, etc)

Reply:
Do you have any suggestion on the better naming?

Reply:
Perhaps call this the "rootWidget"
Not a great name but that would match the inspector tree where we may show a different root widget given a selection than the selection itself. For example, if the selection is not a summary tree node, we will use the nearest summary tree ancestor as the root node.